addons/common/fnc_synchTeamColors.sqf

```diff
@@ -0,0 +1,27 @@
+/* ----------------------------------------------------------------------------
+Internal Function: CBA_fnc_synchTeamColors
+
+Description:
+    Synchs the team colors every second. Does not need to be called manually.
```

Comment:
There is no time delay in this function, but rather in the PFH calling this function. I'd remove the `every second` part from the description.